### PR TITLE
Added more examples and tools

### PR DESCRIPTION
**Pull request by nahmedraja**

### CODE REVIEW

File: cwl/tools/GOEnrichment/GOEnrichment.cwl [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ DockerRequirement:
+    dockerPull: eclipse-temurin:latest
+ InitialWorkDirRequirement:
+    listing:
+      - $(inputs.Jar)
+baseCommand: java
+arguments:
+  - valueFrom: "-jar GOEnrichment.jar"
+    position: 1
+    shellQuote: false
```
Comment (mr-c) [inline cwl/tools/GOEnrichment/GOEnrichment.cwl:17]: I'm surprised that works with the space; also the filename could be different.

I would recommend removing this and adding the follow to the `Jar` input:
``` yaml
inputBinding:
  position: 0
  prefix: "-jar"
```

File: cwl/tools/GOEnrichment/GOEnrichment.cwl [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      location: https://github.com/DanFaria/GOEnrichment/releases/download/v2.0.1/GOEnrichment.jar
+  GOEnrichment_in_1:
+    type: File
+    format: "http://edamontology.org/format_2330" # Textual format
+    inputBinding:
+        prefix: "-s"
+        position: 1
+  GOEnrichment_in_2:
+    type: File
+    format: "http://edamontology.org/format_2196_plain" # OBO format
```
Comment (mr-c) [inline cwl/tools/GOEnrichment/GOEnrichment.cwl:32]: `https://edamontology.org/format_2196_plain` is not in EDAM; did you mean http://edamontology.org/format_2196 (generic OBO format, could be XML or plain text) or its sub-format http://edamontology.org/format_2549 ("OBO ontology text format")?

File: cwl/tools/GOEnrichment/GOEnrichment.cwl [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    inputBinding:
+        prefix: "-g"
+        position: 2
+  GOEnrichment_in_3:
+    type: File
+    format: "http://edamontology.org/format_3475_plain" # TSV_p
```
Comment (mr-c) [inline cwl/tools/GOEnrichment/GOEnrichment.cwl:38]: Same as above: http://edamontology.org/format_3475_plain is not an EDAM term. Did you mean just the regular plain TSV, http://edamontology.org/format_3475 (also known as http://www.iana.org/assignments/media-types/text/tab-separated-values ) ?

File: cwl/tools/Comet/Comet.cwl [hunk 4]
```diff
@@ -10,8 +10,7 @@ requirements:
       - $(inputs.Comet_in_2)
       - $(inputs.Params)
   DockerRequirement:
-    dockerPull: spctools/tpp
-    dockerOutputDirectory: /data
```
Comment (mr-c) [inline cwl/tools/Comet/Comet.cwl:14]: Glad to see that `dockerOutputDirectory` isn't needed anymore

File: cwl/tools/ProteinProphet/ProteinProphet.cwl [hunk 5]
```diff
@@ -4,7 +4,7 @@ baseCommand: "ProteinProphet"
 label: ProteinProphet.cwl
 requirements:
   DockerRequirement:
-    dockerPull: spctools/tpp
+    dockerPull: spctools/tpp:version6.3.3
     dockerOutputDirectory: /data
```
Comment (mr-c) [inline cwl/tools/ProteinProphet/ProteinProphet.cwl:8]: Can ` dockerOutputDirectory: /data` be removed now?

File: cwl/tools/GOEnrichment/GOEnrichment.cwl (hunk 1 above)
Comment (nahmedraja) [reply cwl/tools/GOEnrichment/GOEnrichment.cwl:17]: Thanks Michael very nice suggestion. I will do the same at couple of other places as well.

File: cwl/tools/ProteinProphet/ProteinProphet.cwl (hunk 5 above)
Comment (nahmedraja) [reply cwl/tools/ProteinProphet/ProteinProphet.cwl:8]: yes I have removed it now.

File: cwl/tools/GOEnrichment/GOEnrichment.cwl (hunk 2 above)
Comment (vedran-kasalica) [reply cwl/tools/GOEnrichment/GOEnrichment.cwl:35]: Thank you for pointing this out. The "_p" and "_plain" used to be suffixes introduced by the APE encoding. I removed them here as well.

File: cwl/tools/GOEnrichment/GOEnrichment.cwl (hunk 3 above)
Comment (vedran-kasalica) [reply cwl/tools/GOEnrichment/GOEnrichment.cwl:41]: The same as above.

**Review by vedran-kasalica — APPROVED** [review]
All CWL tool annotations, apart from `mzRecal`, appear to function correctly with the provided CWL workflows. 

After adding the inputs.yml file for workflow 4 and manually providing two inputs (.obo and .gaf) for workflow 5, workflows 1 to 5 run/benchmarked well. `mzRecal` encountered issues in workflow 6 and workflow 7; however, mzRecal can remain unchanged in this pull request since Issue https://github.com/Workflomics/containers/issues/54 was created specifically to address that problem.

**Pull request merged**